astropy/io/fits/tests/test_table.py

```diff
@@ -1016,7 +1016,7 @@ def test_add_data_by_rows(self):
         assert tbhdu.columns.columns[2].array[3] == 'A Note'
         assert (tbhdu.columns.columns[3].array[3] ==
                 np.array([1., 2., 3., 4., 5.], dtype=np.float32)).all()
-        assert tbhdu.columns.columns[4].array[3] == True  # nopep8
+        assert tbhdu.columns.columns[4].array[3] is True  # nopep8
```

Comment:
I have a deja vu that these are `==` for a reason and thus the nopep8?

Reply:
Indeed np bool `True` is not the Python `True` literal, but it is equal `==`.  So this needs to be kept.  So change `nopep8` to `noqa`.

Reply:
Yep, this is fixed, and I changed for noqa as well.